### PR TITLE
Fixing a few errors in the template routing for custom theme overrides

### PR DESCRIPTION
**Pull request by XanderCalvert**
### 🎫 Ticket

[ET-1491] 

### 🗒️ Description

While doing some work with the plug in I noticed that some of the template override locations were wrong and if I can save someone else 20 mins of head scratching that would be worth while.

Just amending comments

Wrong routes mentioned

Did a search through the plugin in VScode so should have gotten them all for you guys

### ✔️ Checklist
- [ ] I've included a changelog entry in the readme.txt file. 
- [ ✔️] My code is tested. 
- [✔️ ] My code has [proper inline documentation](https://the-events-calendar.github.io/products-engineering/docs/code-standards/).

New to pull requests so hope I did it right for you guys

[ET-1491]: https://theeventscalendar.atlassian.net/browse/ET-1491?atlOrigin=eyJpIjoiNWRkNTljNzYxNjVmNDY3MDlhMDU5Y2ZhYzA5YTRkZjUiLCJwIjoiZ2l0aHViLWNvbS1KU1cifQ

### CODE REVIEW

**Comment by juanfra** [comment]
Hi @XanderCalvert - Thanks a lot for your contribution. We're going to be including these changes in the upcoming release. 🎉

File: src/views/blocks/tickets/opt-out-hidden.php [hunk 1]
```diff
@@ -4,7 +4,7 @@
  * Form Opt-Out
  *
  * Override this template in your own theme by creating a file at:
- * [your-theme]/tribe/tickets/blocks/rsvp/form/opt-out.php
+ * [your-theme]/tribe/tickets/blocks/rsvp/form/opt-out-hidden.php
```
Comment (juanfra) [inline src/views/blocks/tickets/opt-out-hidden.php:7]:
```suggestion
 * [your-theme]/tribe/tickets/blocks/tickets/opt-out-hidden.php
```

File: src/views/blocks/tickets/registration/summary/content.php [hunk 2]
```diff
@@ -4,7 +4,7 @@
  * Registration Summary
  *
  * Override this template in your own theme by creating a file at:
- * [your-theme]/tribe/tickets/blocks/tickets/registration/summary.php
+ * [your-theme]/tribe/tickets/blocks/tickets/registration/content.php
```
Comment (juanfra) [inline src/views/blocks/tickets/registration/summary/content.php:7]:
```suggestion
 * [your-theme]/tribe/tickets/blocks/tickets/registration/summary/content.php
```

**Comment by rafsuntaskin** [comment]
@juanfra can you please approve this one? I will add the readme from the other PR: https://github.com/the-events-calendar/event-tickets/pull/2384

**Comment by bordoni** [comment]
@Sjaure can you make sure we get this aligned for the next Maintenance Release?

**Comment by Sjaure** [comment]
Hey @XanderCalvert , circling back to this one; it’s been a bit, but we wanted to drop an update.
We ended up including these fixes in another PR, [#2384]. Thanks again for catching those issues!
I’ll go ahead and close this PR to keep things tidy.

**Pull request closed without merging**